CHANGELOG.md

```diff
@@ -3,6 +3,8 @@
 ## unreleased
 * BraintreeCore
   * Update `endpoint` syntax sent to FPTI for 3D Secure and Venmo flows
+* ThreeDSecure
+  * Update `ThreeDSecureActivity` theme attributes to prevent the Action Bar title from displaying and enforce transparency properly with AppCompat theme attributes
```

Comment:
Just curious if we had an inbound about this that we should link here or if this was just a general bug we noticed?

Reply:
There was mention of it somewhere I think the bug was closed but resurfaced in Android 15. I'll try and find it.

Reply:
This PR is the only thing I could find: https://github.com/braintree/braintree_android/pull/1025.

Reply:
I don't think an actual issue was filed, but I could be wrong.

Reply:
Ah cool, if we don't have one that's fine, was just curious more than anything else.